### ISSUE TITLE
Not work in python 3.4 windows 8

### ISSUE THREAD

**Issue by wannaphongcom** [peterbrittain/asciimatics#15]
I using asciimatics 1.5.0.post1 in python 3.4.3 on Windows 8 64 bit.
code
```python
from asciimatics.effects import Cycle, Stars
from asciimatics.renderers import FigletText
from asciimatics.scene import Scene
from asciimatics.screen import Screen

def demo(screen):
    effects = [
        Cycle(
            screen,
            FigletText("ASCIIMATICS", font='big'),
            screen.height / 2 - 8),
        Cycle(
            screen,
            FigletText("ROCKS!", font='big'),
            screen.height / 2 + 3),
        Stars(screen, 200)
    ]
    screen.play([Scene(effects, 500)])

Screen.wrapper(demo)
```
but It's not working.
```python
Traceback (most recent call last):
  File "1.py", line 20, in <module>
    Screen.wrapper(demo)
  File "C:\py34\lib\site-packages\asciimatics\screen.py", line 478, in wrapper
    func(win_screen)
  File "1.py", line 18, in demo
    screen.play([Scene(effects, 500)])
  File "C:\py34\lib\site-packages\asciimatics\screen.py", line 701, in play
    effect.update(frame)
  File "C:\py34\lib\site-packages\asciimatics\effects.py", line 63, in update
    self._update(frame_no)
  File "C:\py34\lib\site-packages\asciimatics\effects.py", line 180, in _update
    self._screen.centre(line, y, self._colour)
  File "C:\py34\lib\site-packages\asciimatics\screen.py", line 630, in centre
    self.paint(text, x, y, colour, attr, colour_map=colour_map)
  File "C:\py34\lib\site-packages\asciimatics\screen.py", line 654, in paint
    self.print_at(text, x, y, colour, attr, bg, transparent)
  File "C:\py34\lib\site-packages\asciimatics\screen.py", line 962, in print_at
    self._double_buffer[y][x + i] = (c, colour, attr, bg)
TypeError: list indices must be integers, not float
```
Thank you. Merry Christmas.

**Comment by wannaphongcom** [comment]
I can solve the problem above, using int().
Please edit the sample code.
```python
from asciimatics.effects import Cycle, Stars
from asciimatics.renderers import FigletText
from asciimatics.scene import Scene
from asciimatics.screen import Screen

def demo(screen):
    effects = [
        Cycle(
            screen,
            FigletText("ASCIIMATICS", font='big'),
            int(screen.height / 2 - 8)),
        Cycle(
            screen,
            FigletText("ROCKS!", font='big'),
            int(screen.height / 2 + 3)),
        Stars(screen, 200)
    ]
    screen.play([Scene(effects, 500)])

Screen.wrapper(demo)
```

**Comment by peterbrittain** [comment]
This is a difference between python 2 and 3.  I've managed to find all of them in the samples, but clearly not the docs!

I'll fix up the reader as per your changes.  Thanks and merry Christmas!

**Comment by peterbrittain** [comment]
Fixed in master branch.  Should be released in V1.6.
